From 0000000000000000000000000000000000000001 Mon Sep 17 00:00:00 2001
From: mpv-build <mpv-build@users.noreply.github.com>
Date: Tue, 16 Sep 2025 21:27:25 +0800
Subject: [PATCH] mpv: copy mpv-register and mpv-unregister helpers

---
 win/packages/mpv.cmake | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/win/packages/mpv.cmake b/win/packages/mpv.cmake
index 874ee95..1014cc7 100644
--- a/win/packages/mpv.cmake
+++ b/win/packages/mpv.cmake
@@ -66,6 +66,8 @@ ExternalProject_Add_Step(mpv copy-binary
     DEPENDEES strip-binary
     COMMAND ${CMAKE_COMMAND} -E copy <BINARY_DIR>/mpv.exe                           ${CMAKE_CURRENT_BINARY_DIR}/mpv-package/mpv.exe
     COMMAND ${CMAKE_COMMAND} -E copy <BINARY_DIR>/mpv.com                           ${CMAKE_CURRENT_BINARY_DIR}/mpv-package/mpv.com
+    COMMAND ${CMAKE_COMMAND} -E copy <SOURCE_DIR>/etc/mpv-register.bat              ${CMAKE_CURRENT_BINARY_DIR}/mpv-package/mpv-register.bat
+    COMMAND ${CMAKE_COMMAND} -E copy <SOURCE_DIR>/etc/mpv-unregister.bat            ${CMAKE_CURRENT_BINARY_DIR}/mpv-package/mpv-unregister.bat
     COMMAND ${CMAKE_COMMAND} -E copy <BINARY_DIR>/mpv.pdf                           ${CMAKE_CURRENT_BINARY_DIR}/mpv-package/doc/manual.pdf
     COMMAND ${CMAKE_COMMAND} -E copy ${MINGW_INSTALL_PREFIX}/etc/fonts/fonts.conf   ${CMAKE_CURRENT_BINARY_DIR}/mpv-package/mpv/fonts.conf
     ${mpv_copy_debug}
-- 
2.52.0
